src/v/features/feature_table.cc

```diff
@@ -152,14 +152,20 @@ std::string_view to_string_view(feature_state::state s) {
     __builtin_unreachable();
 };
 
+namespace {
+
 // The version that this redpanda node will report: increment this
 // on protocol changes to raft0 structures, like adding new services.
-static constexpr cluster_version latest_version = cluster_version{14};
+constexpr cluster_version latest_version = to_cluster_version(
```

Comment:
The `static` was removed because the global is moved into an anonymous namespace instead. I guess I could drop that and just use `constexpr` as it provides internal linkage.